src/System.Private.ServiceModel/tests/Scenarios/Binding/Http/BasicHttpBindingTests.cs

```diff
@@ -4,17 +4,20 @@
 using System;
 using System.ServiceModel;
 using System.ServiceModel.Channels;
+using System.ServiceModel.Tests.Common;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using TestTypes;
 using Xunit;
 
-public static class Binding_Http_BasicHttpBindingTests
+public class Binding_Http_BasicHttpBindingTests : OuterLoopTests
```

Comment:
If the class fixture is not available in .Net native I think there is still an initialization story with the constructor and dispose method, which is the preferred method for xUnit. At least running the tests with corerun that part works. I used a base class so that I don't have to add a constructor/dispose method to each test. But I have no problem doing that. I'll find out if the class fixture works at all.